src/firebase/CRUDServices/imageService.js

```diff
@@ -28,68 +28,39 @@ export default class ImageService {
      * @param {String} userId user id.
      * @param {String} experimentId experiment id.
      * @param {File} image self image file.
-     * @return {Boolean} succeed or not.
+     * @return {String} succeed or not.
      */
-    postImage = async (userId, experimentId, image) => {
+    postRawImage = async (userId, experimentId, image) => {
       const folderPath = userId + '-' + experimentId + '/';
```

Comment:
nit:
```
      const folderPath = `${userId}-${experimentId}/`;
```

eh after writing this out, I feel like this is equally ugly lol don't mind me here

Reply:
lol, the string literal looks shorter.